samples/Playground/Playground/Playground/Playground.Shared/App.xaml.cs

```diff
@@ -275,7 +275,8 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
 		private static void RegisterRoutes(IViewRegistry views, IRouteRegistry routes)
 		{
 			views.Register(
-						new ViewMap(ViewModel: typeof(ShellViewModel)),
+						new ViewMap<ShellView,ShellViewModel>(),
+						//new ViewMap(ViewModel: typeof(ShellViewModel)),
```

Comment:
remove?

Reply:
I've left that there for testing - specifying the ShellView is optional, so was good to test with both options. Let me add a note in there for this